warehouse/malware/checks/base.py

```diff
@@ -10,18 +10,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from warehouse.malware.models import MalwareCheck, MalwareCheckState
+from warehouse.malware.models import MalwareCheck, MalwareCheckState, MalwareVerdict
 
 
 class MalwareCheckBase:
     def __init__(self, db):
         self.db = db
         self._name = self.__class__.__name__
         self._load_check()
+        self._verdicts = []
+
+    def add_verdict(self, verdict):
```

Comment:
What do you think about making `verdict` into a kwargs instead of a direct dictionary? That would make it consistent with the `record_event` API.

Reply:
Sure!